### ISSUE TITLE
[nextgen-editor] Ul mssing in child elements

### ISSUE THREAD

**Issue by xf-** [getgrav/grav-premium-issues#123]
Markdown output from editor  (here using ui tabs, but also with other elements)
```markdown
[ui-tabs]
[ui-tab]
*   fgfgfg
*   dfg
[/ui-tab]
[ui-tab]
*   fdg
*   dfg
[/ui-tab]
[/ui-tabs]
```

Code
![image](https://user-images.githubusercontent.com/1295633/124025509-cf67d900-d9f0-11eb-9f64-d5967e52dac9.png)

Temp fix is expert mode and add an empty line after the last list item, but the next save with nextgen-editor will remove the line and the issue is back.

**Comment by rhukster** [comment]
I have a couple of questions for you:

1. is that markdown example exactly how it looks in the .md file?
2. What is your `shortcode-core` parser configure to use? It should use "regular" parser.

**Comment by xf-** [comment]
1. yes, c&p from md file
2. All of them. Switched around to maybe fix it.

**Comment by xf-** [comment]
i tried to debug the issue with youtube shortcode first and saw your comment in https://github.com/thunderer/Shortcode/issues/65

[youtube autoplay="1" rel="0"]https://www.youtube.com/watch?v=YGgdfrkBlaU[/youtube]
https://www.youtube.com/watch?v=YGgdfrkBlaU[/youtube]

I think in general the trim is an issue, because it breaks workarounds a list wants a empty line markdown to next element and the shortcodes are always nextline.

**Comment by w00fz** [comment]
I think this is a shortcode issue, the rendered shortcode missing the `<ul>` wrapper around the second tab, new lines etc, should be handled by Shortcode and not NextGen.

**Comment by rhukster** [comment]
I will investigate this shortly.

**Comment by xf-** [comment]
@rhukster thx

For me, it sounds like a bug in new editor, because it touches empty lines and that is often used in Markdown to separate stuff. Markdown from Old editor will break in new editor.

More or less 2 Bugs
- Wrong rendering -> properly short code plugin
- Adding an empty line after a list, so the list is standalone and markdown stops - Editor removes it is from md file.

**Comment by w00fz** [comment]
hey @xf- 
I fixed the issue by enforcing an extra empty line before and after the content of shortcodes that are nested (`[ui-tab]`, `[ui-accordion]`) when saving.

This wasn't really an issue with NextGen Editor per se, so you will find this fix in the v5.0.7 of Shortcode Core plugin.

Cheers,

**Comment by xf-** [comment]
@w00fz in my eyes an issue from the editor, because it changes the source code, compared to old one. But it is a solution.
